docs/data-availability/light-client.md

```diff
@@ -3,8 +3,31 @@ id: light-client
 title: Light Client
 ---
 
+# NEAR light client
+
 A trustless off-chain light client for NEAR with DA-enabled features, such as KZG commitments, Reed-Solomon erasure coding & storage connectors.
 
+:::note
+The NEAR Light client was inspired by the [Helios light client](https://github.com/a16z/helios), where there were once many light client implementations for Ethereum until it was released. 
+:::
```

Comment:
I'd say we probably don't need this in the docs, we can keep it in the README.

Reply:
👍 removed